tests/components/nut/test_config_flow.py

```diff
@@ -21,7 +31,25 @@ def _get_mock_pynutclient(list_vars=None):
     assert result["type"] == "form"
     assert result["errors"] == {}
 
-    mock_pynut = _get_mock_pynutclient(list_vars={"battery.voltage": "voltage"})
+    mock_pynut = _get_mock_pynutclient(
+        list_vars={"battery.voltage": "voltage"}, list_ups=["ups1"]
+    )
+
+    with patch(
+        "homeassistant.components.nut.PyNUTClient", return_value=mock_pynut,
+    ):
+        result2 = await hass.config_entries.flow.async_configure(
+            result["flow_id"],
+            {
+                "host": "1.1.1.1",
+                "username": "test-username",
+                "password": "test-password",
+                "port": 2222,
+            },
+        )
+
+    assert result2["step_id"] == "resources"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5329694)